scripts/output_env.sh

```diff
@@ -23,11 +23,15 @@ print_version()
     shift
     ARGS="$1"
     shift
-    FAIL_MSG="$1"
+    VARIANT=$1
     shift
 
+    if [ ! -z $VARIANT ]; then
+        VARIANT=" ($VARIANT)"
+    fi
+
     if ! `type "$BIN" > /dev/null 2>&1`; then
```

Comment:
Preexisting:
```suggestion
    if ! type "$BIN" > /dev/null 2>&1; then
```

`` `foo >dev/null` `` takes the output of `foo >/dev/null`, which is always empty, and (after certain expansions) executes it as a command. The empty command does nothing and has the same return status as the last command substitution performed as part of the expansions done on the command line. So this happens to work, but it's convoluted.